store/postgres/src/connection_pool.rs

```diff
@@ -503,8 +503,16 @@ impl ConnectionPool {
     /// # Panics
     ///
     /// If any errors happen during the migration, the process panics
-    pub fn setup(&self) {
-        self.get_ready().ok();
+    pub async fn setup(&self) {
+        let pool = self.clone();
+        graph::spawn_blocking({
```

Comment:
Lets use `spawn_blocking_allow_panic` here, `graph::spawn_blocking` is really an antipattern since futures should not block.

Reply:
Changed it to use `spawn_blocking_allow_panic` and panicking if `setup` panics